lib/src/chewie_player.dart

```diff
@@ -495,11 +495,14 @@ class ChewieController extends ChangeNotifier {
   /// Defines a custom RoutePageBuilder for the fullscreen
   final ChewieRoutePageBuilder? routePageBuilder;
 
-  static ChewieController of(BuildContext context) {
+  static ChewieController of<ChewieControllerProvider>(BuildContext context) {
```

Comment:
`ChewieControllerProvider` is not a Generic Class. It is a specific class used to pass the `ChewieController` to any descendant widget. It should not be used as a generic here.